docs/src/main/asciidoc/security-oidc-bearer-token-authentication.adoc

```diff
@@ -1339,11 +1345,11 @@ xref:security-openid-connect-multitenancy.adoc#tenant-config-resolver[Dynamic te
 Authentication that requires a dynamic tenant will fail.
 ====
 
-[[oidc-request-filters]]
+[[oidc-request-filters-a]]
 == OIDC request filters
 
 You can filter OIDC requests made by Quarkus to the OIDC provider by registering one or more `OidcRequestFilter` implementations, which can update or add new request headers, and log requests.
-For more information, see xref:security-oidc-code-flow-authentication#oidc-request-filters[OIDC request filters].
+For more information, see xref:security-oidc-code-flow-authentication#oidc-request-filters-b[OIDC request filters].
```

Comment:
Why `b` and not a ?

Reply:
This cross reference in `security-oidc-bearer-token-authentication.adoc` points to the section in `security-oidc-code-flow-authentication.adoc` whose ID is `oidc-request-filters-b`.

Search results for "oidc-request-filters":

```
/asciidoc/security-oidc-bearer-token-authentication.adoc:
  1348:  [[oidc-request-filters-a]]
  1352:  information, see xref:security-oidc-code-flow-authentication#oidc-request-filters-b[OIDC request filters].


/asciidoc/security-oidc-code-flow-authentication.adoc:
  311:  [[oidc-request-filters-b]]


/asciidoc/security-openid-connect-client-reference.adoc:
  1073:  [[oidc-request-filters-c]]
```

Reply:
I'm sorry Rolfe, but I'm confused, perhaps we can try what Michal suggested below ?

Reply:
That’s perfectly understandable, @sberyozkin. These IDs can trip up many people. Let’s work on making it clearer. I’ve applied @michalvavrik's suggestion to the current PR. Please take a look and let me know if I can make any further tweaks to improve clarity.